### PR TITLE
Fix e2e test page selectors due to hds refactor

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
# Description

Refactor test page selectors to accommodate changes made to app state messages.
Upgrade playwright to newest version in order to install newer browser versions since `popover` used in newer hds version isn't supported by firefox v124.

## Screenshots (if appropriate)
tests passing in ce
image

tests passing in ent, the tests failing are flaky (there's already a ticket for that)
image

## How to Test

1. build boundary binary (using most recent ui commit in main)
2. follow readme instructions in `e2e-testing` folder to setup enos testing env.
3. run tests & validate only tests failing are in firefox & webkit browsers.

## Checklist

- [ ] I have added before and after screenshots for UI changes
- [ ] I have added JSON response output for API changes
- [ ] I have added steps to reproduce and test for bug fixes in the description
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by ZedLi — APPROVED** [review]
Looks good, ran fine against ent tests!

File: ui/admin/package.json [hunk 1]
```diff
@@ -53,7 +53,6 @@
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "@hashicorp/ember-asciinema-player": "https://github.com/hashicorp/ember-asciinema-player.git#e047a096039cff70234c232efe75dcad74c6358a",
-    "@playwright/test": "^1.43.0",
```
Comment (calcaide) [inline ui/admin/package.json:56]: Good catch deleting the playwright deps from ui/admin and ui/desktop! 🙌

**Review by calcaide — APPROVED** [review]
Nice refactor @lisbet-alvarez ! Awesome work!!

I ran the CE edition as expected, without an issue.

I did encounter 1 failing test within ent which does not match with the one you reported within PR description.
Not sure if related to this PR, but leaving an screenshot here. If you think its not related, please let me know I will proceed to debug 😉 
Screenshot 2024-12-09 at 3 53 56 PM

**Comment by lisbet-alvarez** [comment]
> Nice refactor @lisbet-alvarez ! Awesome work!!
> 
> I ran the CE edition as expected, without an issue.
> 
> I did encounter 1 failing test within ent which does not match with the one you reported within PR description. Not sure if related to this PR, but leaving an screenshot here. If you think its not related, please let me know I will proceed to debug 😉 Screenshot 2024-12-09 at 3 53 56 PM

hm i don't think its related, those tests didn't fail for me at all.

**Comment by calcaide** [comment]
> > Nice refactor @lisbet-alvarez ! Awesome work!!
> > I ran the CE edition as expected, without an issue.
> > I did encounter 1 failing test within ent which does not match with the one you reported within PR description. Not sure if related to this PR, but leaving an screenshot here. If you think its not related, please let me know I will proceed to debug 😉 Screenshot 2024-12-09 at 3 53 56 PM
> 
> hm i don't think its related, those tests didn't fail for me at all. thank you for testing as well!

@lisbet-alvarez 
Gotcha! I will review and debug that test 😉  Thanks for the feedback!!

**Review by cameronperera — APPROVED** [review]

**Pull request merged**